### PR TITLE
[54|70] Create validate-build.yml

### PR DESCRIPTION
**Pull request by arkrusade**

### CODE REVIEW

**Review by xumaple — CHANGES_REQUESTED** [review]
Can we validate the backend as well if any files in cms/ have changed? Can do an npm run build there as well I think...

File: .github/workflows/validate-build.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Validate Frontend Build
+run-name: Validate Frontend Build (kicked off by ${{ github.actor }})
+on:
+  pull_request:
+    branches:
+      - "main"
```
Comment (xumaple) [inline .github/workflows/validate-build.yml:6]: Does this cause the build to be rerun for every repush to a PR, or only on its first creation?

Comment (arkrusade) [reply .github/workflows/validate-build.yml:6]: On create and on new commits. Found the [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request): 

> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened

**Comment by arkrusade** [comment]
@thedomcab I had to change the `cms/package.json` in order to validate it, since it would always fail `npm install` otherwise (this undid your audit fixes  from earlier).

**Comment by thedomcab** [comment]
> @thedomcab I had to change the `cms/package.json` in order to validate it, since it would always fail `npm install` otherwise (this undid your audit fixes from earlier).

okay. i just need to run docker and see what happen from this branch of yours. will look into today.

**Comment by thedomcab** [comment]
looks all good from running docker on my mac.

**Review by xumaple — APPROVED** [review]

**Review by xumaple — APPROVED** [review]

**Pull request merged**